src/Nelmio/Alice/Instances/Processor/Processable.php

```diff
@@ -45,6 +45,12 @@ public function getValue()
     public function valueMatches($regex)
     {
         if (preg_match($regex, $this->value, $matches)) {
+            $matches       = array_map(
```

Comment:
if the preg_match change is deprecated I would not change that yet, it wouldn't be the first time such things are reverted and I didn't see much discussion in https://github.com/php/php-src/pull/1303.